src/scaffoldmaker/meshtypes/meshtype_3d_stellate1.py

```diff
@@ -184,11 +184,13 @@ def generateBaseMesh(cls, region, options):
         # arm group annotations for user
         armTerms, _ = getAutomaticArmFaceTerms(armCount)
         armGroups = [AnnotationGroup(region, armTerm) for armTerm in armTerms]
-        stellateGroup = AnnotationGroup(region, get_stellate_term("cervicothoracic ganglion"))
-        annotationGroups = [stellateGroup] + armGroups
+        wholeScaffoldGroup = AnnotationGroup(region, ("stellate", None))
+        if isMouse:
+            wholeScaffoldGroup = AnnotationGroup(region, get_stellate_term("cervicothoracic ganglion"))
```

Comment:
I don't like needlessly making an object then remaking it (can be expensive in general) so prefer:
```
wholeScaffoldTerm = get_stellate_term("cervicothoracic ganglion") if isMouse else ("stellate", None)
wholeScaffoldGroup = AnnotationGroup(region, wholeScaffoldTerm)
```
Also, suggest rename these back to stellateTerm and stellateGroup, so consistent with stellateMeshGroup later. (Alternatively rename the latter to wholeScaffoldMeshGroup for consistency).